### ISSUE TITLE
Aggiunta testimonianza DataMining-2025-Pindilli Lorenzo

### ISSUE THREAD

**Issue by lollo3393** [UnicalLoveTelegram/IndiceArgomentiOrale#28]
#  ingegneria informatica LT

#### 2024/2025

- Pindilli Lorenzo
    - Principio a priori
        - Algoritmo  a priori
    - Kmeans (con tutte le formule)
        -Fuzzy Kmeans
        -Bisect Kmeans
    - DbScan
    - Clustering gerarchico
        -Clustering aagglomerativo (algoritmo e spiegazione e complessità)
        -Clustering divisivo (Algortimo spiegazione e complessità)
    - Reti convoluzionali
    - Gini index
    - Canali nelle reti convoluzionali
    - regressione Lineare
    - Regressione Logistica
    - Metriche negli alberi decisionali (Accuracy precision e recall)
    - Entropia
    - Teorema universale delle reti neurali
    - Kmeans++
    - Knn 
    - Classificatori bayesiani
    - Discesa del gradiente
    - Discesa del gradiente stocastico
    - Xor problem, con grafici 
    - Embedding 
    - Output in una rete shallow (dimensione)
    - Backpropagation 
    - In generale chiede tutto e lo vuole con precisione, consiglio di fare al meglio tutti gli algoritmi che ci sono scritti, e di fare la complessità
    - Quali sono le funzioni di attivazione più usate e fare il disegno (gradino, softMax, ecc..)
    - Misure di similarità per i cluster 
    - come comportarsi se un cluster diventa "vuoto"
    - tecniche di Ensable
